cdc/kv/client.go

```diff
@@ -63,6 +63,11 @@ const (
 
 	// The threshold of warning a message is too large. TiKV split events into 6MB per-message.
 	warnRecvMsgSizeThreshold = 12 * 1024 * 1024
+
+	// hard code switch
+	// true: use kv client v2, which has a region worker for each stream
+	// false: use kv client v1, which runs a goroutine for every single region
+	kvClientV2 = true
```

Comment:
```suggestion
	kvClientV2 = false
```
turn off before PR is merged